tutorial.js

```diff
@@ -0,0 +1,78 @@
+var index = 0;
+var difficulty = localStorage.getItem('difficulty');
+
+$(document).ready(function() {
+
+$("#btn").click(function() {
+    
+    if(index == 0)
+    {
+        var tmp = $("<div/>");
+        tmp.addClass("btn2");
+        
+        var tmp2 = $("<p/>");
+        tmp2.text("괜찮습니다!");
+        tmp.append(tmp2);
+        $("#text").append(tmp);
+        $(".btn2").click(function() {
+            if(difficulty == 1)
+            location.href="level1.html?level_info=1"
+            else if(difficulty == 2)
+            location.href="level2.html?level_info=2"
+            else
+            location.href="level3.html?level_info=3"
```

Comment:
아래와 같이 한 줄로 줄일 수도 있을 듯하네요
```suggestion
            location.href = `level${difficulty}.html?level_info=${difficulty}`;
```